docs/features/spritesvg.md

```diff
@@ -0,0 +1,29 @@
+It uses [gulp.svg-sprite](https://github.com/jkphl/gulp-svg-sprite) and [gulp.svgmin](https://github.com/ben-eb/gulp-svgmin)
+
+
+## Usage
+
+- Add your raw svg icon in `config.spritesvg.src`  
+- run `gulp spritesvg`  
+- A  `sprite.svg` file will be generated as a collection of symbol  
+- Implement it with `<use>` [Example](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use)  
+ - Drupal 8 and twig implementation[ Example](https://www.lullabot.com/articles/better-svg-sprite-reuse-in-drupal-8)  
+ - [check if you need svg4everybody](https://github.com/jonathantneal/svg4everybody#implementation-status)
+
+Notes:
+- symbol IDs are prefixed with `icon-` and append file name  
+- get your designer to make icons monochrome and convert the strokes into paths, also simplify the paths down to single shapes.[Example](https://medium.com/sketch-app-sources/preparing-and-exporting-svg-icons-in-sketch-1a3d65b239bb)  
+
```

Comment:
Could you please remove extra empty lines?

Reply:
done